### PR TITLE
Week 1

### PR DESCRIPTION
**Pull request by astrobliss**
Completed work on week-1 material.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -20,9 +29,48 @@ function addRandomGreeting() {
...
+    {description:"The Repository of a react project I was on", url: "https://github.com/astrobliss/Cadabra"},
+    {description:"A website that makes trump rap, entertainment purposes only", url: "https://astrobliss.github.io/trump-pad/"}
+  ];
+
+  const link = getRandomElement(links);
+
+  linkElement = document.getElementById('link-container');
+  linkElement.text = link.description;
+  linkElement.href = link.url;
+}
```
Comment (chrisachille) [inline portfolio/src/main/webapp/script.js:76]: Small think, but can you make sure all files end in a new line.  It helps when looking at other changes later on, and just makes the file a little cleaner.  This also applies for style.css.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -2,8 +2,25 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  font-family: 'Source Serif Pro';
```
Comment (chrisachille) [inline portfolio/src/main/webapp/style.css:5]: Might be worth alphabetizing the sections, so it is a little cleaner when adding more.

**Review by chrisachille — APPROVED** [review]
Couple small comments, but it looks good to me.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (astrobliss) [reply portfolio/src/main/webapp/script.js:76]: I'll make sure to do that. Is this StackOverflow post about it telling the full story?
https://stackoverflow.com/a/729795

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (astrobliss) [reply portfolio/src/main/webapp/style.css:5]: I'll put the change in

**Pull request merged**